moncli/types.py

```diff
@@ -211,4 +211,24 @@ def validate_timezone(self, value):
         try:
             pytz.timezone(value)
         except (UnknownTimeZoneError):
-            raise ValidationError('Unknown time zone "{}".'.format(value))
\ No newline at end of file
+            raise ValidationError('Unknown time zone "{}".'.format(value))
+
+class DependencyType(MondayType):
+
+    native_type = list
+    native_default = []
+    null_value = {}
+
+    def _process(self, value):
+        value_data = None
+        value_list = []
+        try:
```

Comment:
Also, this is not working.  You forgot to add the cast to int.

Something like this would be cleanest.
```python
value_list = []
for data in value:
    try:
        value_list.append(int(data))
    except ValueError
        raise ConversionError('Invalid item ID: "{}".'.format(data))
return value_list
```